android/fastlane/Appfile

```diff
@@ -0,0 +1,2 @@
+json_key_file("") # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
```

Comment:
Strange, seems like we don't even have this in integreat.

Reply:
This is fine if we only deploy one app here. This avoid setting it in the Fastfile

Reply:
oke so i keep it? What about you comment above?